### ISSUE TITLE
Associate a label to all fields

### ISSUE THREAD

**Issue by KittyGiraudel** [morkro/happy-plants#44]
All visible fields should have an associated label to be accessible for assistive technologies. In the category section for instance, the label wraps the field without providing any content. Consider not wrapping the input element, and have a proper label attached to it. Feel free to visually hide the label if you think it’s getting in the way.

Edit: the water input also doesn’t have a label.

**Comment by morkro** [comment]
Closing this as work on version 2 will cover this.
